go.mod

```
@@ -92,6 +92,8 @@ require (
 	gopkg.in/yaml.v2 v2.4.0
 )
 
+replace github.com/minio/pkg => ../minio-pkg
```

You can replace with your branch/commit number. Do:
```suggestion
replace github.com/minio/pkg => github.com/donatello/minio/pkg b0deda1e5950e03890c8ac410a5a699a16f742e5
```

And `go mod tidy` and you can push a working build

---

Done, thanks!